_includes/selector.html

```diff
@@ -371,7 +371,7 @@
 
             // all possible values
             python_vers: ["3.9", "3.10", "3.11"],
-            cuda_vers: ["11.2", "11.8", "12.0", "12.2"],
+            cuda_vers: ["11.2", "11.8", "12.0", "12.2", "12.5"],
```

Comment:
```suggestion
            cuda_vers: ["11.4-11.8", "12.0-12.2", "12.0-12.5"],
```

Reply:
I like this idea _but_ I don't think we need to have both `12.0-12.2` and `12.0-12.5`. We should have only one CUDA 12 range.

Reply:
Yeah the only reason there is both now is one range is for nightlies and one is for stable (since the two are handled differently atm). If we did this after the release, we could just have one